src/ui/components/ReactPanel.tsx

```diff
@@ -383,118 +390,42 @@ function ReactQueuedRenderListItem({
   );
 }
 
-type CommittedRenderProps = CommonProps & {
-  commitPoint: TimeStampedPoint;
-};
-
-function ReactCommittedRenderListItem({
-  currentTime,
-  commitPoint,
-  executionPoint,
-  onSeek,
-}: CommittedRenderProps) {
-  const { point, time } = commitPoint;
-  const isPaused = time === currentTime && executionPoint === point;
-
-  const onClickSeek = () => {
-    onSeek(point, time);
-  };
-
-  return (
-    <>
-      <div
-        className={classnames(styles.eventRow, "group block w-full", {
-          "text-lightGrey": currentTime < time,
-          "font-semibold text-primaryAccent": isPaused,
-        })}
-        onClick={onClickSeek}
-      >
-        <div className="flex flex-row items-center space-x-2 overflow-hidden">
-          <MaterialIcon iconSize="xl">check_circle_outline</MaterialIcon>
-          <Label>React Render Committed</Label>
-        </div>
-      </div>
-    </>
-  );
-}
-
 export function ReactPanel() {
   const dispatch = useAppDispatch();
   const currentTime = useAppSelector(getCurrentTime);
   const executionPoint = useAppSelector(getExecutionPoint);
   const { rangeForDisplay: focusRange } = useContext(FocusContext);
   const [renderDetails, setRenderDetails] = useState<RenderAnalysisResults | null>(null);
 
-  const handleClick = async () => {
-    console.log("Focus range: ", focusRange);
-    const renderAnalysisResults = (await dispatch(findQueuedRendersForRange(focusRange!))) ?? null;
-    setRenderDetails(renderAnalysisResults);
-  };
+  useEffect(() => {
+    setRenderDetails(null);
```

Comment:
:hand: one issue here is that this ends up re-fetching the values if you click to another tab and back, because `<Offscreen>` re-runs effects.

These really _ought_ to be cached somehow.  We can skip it for this PR.  A stopgap follow-up would be to use our existing cache and just do it for the focus range, without worrying about solving the over-fetching problem.

Reply:
yep - any caching will do